client/my-sites/checkout/src/lib/get-jetpack-product-features.ts

```diff
@@ -201,6 +217,14 @@ export default function getJetpackProductFeatures(
 		];
 	}
 
+	if ( isJetpackGrowthPlan( product.product_slug ) ) {
+		return [
+			...getFeatureStrings( 'growth', translate ),
+			...getFeatureStrings( 'social-basic', translate ),
+			...getFeatureStrings( 'support', translate ),
+		];
+	}
+
 	if ( isJetpackScanSlug( product.product_slug ) ) {
 		return [
 			...getFeatureStrings( 'scan', translate ),
```

Comment:
No worries if you can't get to it in this PR, but I noticed that the new Jetpack Social plan (Jetpack Social V1) is not being accounted for here and no features are being shown. It should be pretty easy to add a check for that but I am not sure if all the features are exactly the same 🙃 

Might leave off for now and we can try and see if the social team is aware of this.

FYI @Automattic/jetpack-reach, we have this file that shows features for each product. We had a set of features shown for Social Basic and Social Advanced, we are unsure whether or not Social V1 has the same features or some combination of both, was wondering if you could let us know if we can just use the old feature copy we had before. You can see that [here](https://github.com/Automattic/wp-calypso/blob/ccf6fb5b29ceebbaa4216f39f07b1fce0c1d6a64/client/my-sites/checkout/src/lib/get-jetpack-product-features.ts#L107)

Reply:
Yeah, "Social V1" is a combination of Basic and Advanced plans.

Reply:
Ok so we can just combine that list and display it for the new plan, thank you for confirming!

Reply:
Updated :)